migration-roadmap/search.md

```diff
@@ -124,28 +106,93 @@
 
 </details>
 
-## Visual Comparison
+### SWC
 
-**Legacy Component:**
+<details>
+<summary>Attributes</summary>
 
-<!-- Screenshot of legacy component will be added here -->
+- `action` (string)
+- `label` (string) - defaults to "Search"
+- `method` (string) - "get" | "post" | "dialog"
+- `placeholder` (string) - defaults to "Search"
+- `holdValueOnEscape` (boolean)
+- `size` (inherited from Textfield)
+- `disabled` (inherited from Textfield)
+- `invalid` (inherited from Textfield)
+- `value` (inherited from Textfield)
 
-**Spectrum 2 Component:**
+</details>
 
-<!-- Screenshot of Spectrum 2 component will be added here -->
+<details>
+<summary>Slots</summary>
+
+- `help-text` - default or non-negative help text to associate to your form element
+- `negative-help-text` - negative help text to associate to your form element when `invalid`
+
+</details>
+
+## Comparison
 
-## DOM Structure Changes
+### DOM Structure changes
 
-**Legacy (main branch):**
+<details>
+<summary>Spectrum Web Components:</summary>
 
 ```html
-<form
-    class="spectrum-Search spectrum-Search--size{size} spectrum-Search--quiet"
->
+<sp-search>
+    #shadow-root
+    <form
+        action=""
+        id="form"
+        method=""
+        @submit="handleSubmit"
+        @reset="reset"
+        @keydown="handleKeydown"
+    >
+        <sp-icon-search
+            size=""
+            class="icon magnifier icon-workflow icon-search"
+        ></sp-icon-search>
+        <div id="textfield">
```

Comment:
To go with my other comment about wrapping the form, this `#textfield` div is not around just the `<input>`.